provider/vsphere/environ_broker.go

```diff
@@ -57,8 +57,12 @@ func modelFolderName(modelUUID, modelName string) string {
 
 // vmdkDirectoryName returns the name of the datastore directory in which
 // the base VMDKs are stored for the controller.
-func vmdkDirectoryName(controllerUUID string) string {
-	return fmt.Sprintf("juju-vmdks/%s", controllerUUID)
+func vmdkDirectoryName(parentfolder string, controllerFolderName string) string {
```

Comment:
Could you rename this to `templateDirectoryName` please? It's drifted a long way from being where vmdks are stored. :)